src/worker/storage.ts

```diff
@@ -42,8 +51,20 @@ export class PredictiveTextStudioDexie extends Dexie {
        * +------------------+
        */
       files: "++id, name, wordlist",
+      /**
+       * packageInfo Table Scehma
+       * packageInfor stores the information needed for create the kmp package.
+       * we might also store language name, moded id in the future.
```

Comment:
```suggestion
       * packageInfo stores the information needed for creating  the KMP package.
       * we might also store language name, model ID in the future.
```

Is spell-checking turned on in your VSCode?